### ISSUE TITLE
ERROR - Config file not found at /configs/memgpt_hosted.json   [pip install pymemgpt 03.14]

### ISSUE THREAD

**Issue by quantumalchemy** [letta-ai/letta#1341]
**Describe the bug**
ERROR:
Config file not found at /usr/local/lib/python3.11/site-packages/memgpt/cli/../../configs/memgpt_hosted.json

**Please describe your setup**
  - `pip install pymemgpt`
- [ ] Describe your setup
linux (docker)
files not being installed for pip install
ok in local devel.  install e .

**Comment by aBaldoqui** [comment]
I'm experiencing the same problem.
There was a bug closed regarding this yesterday: https://github.com/cpacker/MemGPT/issues/1320.

**Comment by quantumalchemy** [comment]
works with Devel mode:
git clone https://github.com/cpacker/MemGPT.git
pip install -e .

**Comment by MEllis-github** [comment]
Same error for macOS running
```
pip install -U pymemgpt
memgpt quickstart --backend memgpt
```

Not triggered with 
```
pip install -e .
memgpt quickstart --backend memgpt
```

Git branch: `main`
Git revision: `e8eed7b`

**Comment by datashaman** [comment]
I've got it working by using the code on branch `tutorials` which includes a fix for this.

MacOS doesn't like installing python packages in the system folders, and I don't want to be in a _virtualenv_ whenever I use this.

`pipx` to the rescue. The following got me up and running:
```
pipx install git+https://github.com/cpacker/MemGPT.git@tutorials
```

**Comment by twilwa** [comment]
> I've got it working by using the code on branch `tutorials` which includes a fix for this.
> 
> MacOS doesn't like installing python packages in the system folders, and I don't want to be in a _virtualenv_ whenever I use this.
> 
> `pipx` to the rescue. The following got me up and running:
> 
> ```
> pipx install git+https://github.com/cpacker/MemGPT.git@tutorials
> ```

Will open my own ticket, but I was able to get this up and running out of a desire to use pipx as opposed to making a venv/local folder. Ran into the same problem, but the tutorials branch did fix it -- however, I'm running into a different issue. When I use the --stream flag, I get repeated attempts that don't involve the thinking step, when I don't I get inifinite loop on retries until it times out for too many attempts. Additionally, the persona and human profile information is incorrect in the terminal, but correct when interacting with the agent. maybe helpful info on this ticket
![image](https://github.com/cpacker/MemGPT/assets/111034762/77a4b6eb-fc81-4a4d-bf87-359caadcd7d4)
